docs/How/using_project_mu.md

```diff
@@ -0,0 +1,93 @@
+# How to setup a new Repo for a Platform that will use Project MU?
```

Comment:
We should think through how this doc ties in with https://microsoft.github.io/mu/CodeDevelopment/prerequisites/#workspace-virtual-environment-setup-process

Also, for consistency, is this simply a "Repo", or a "workspace" as the "Tools and Prereq" page calls it?  This is a special repo representing a platform/product which we will call a workspace?

Reply:
added #0, helpful?

Reply:
It helps.  Perhaps we could also define "Root workspace" and contrast them?  When might they be different?

Reply:
added a sentance